src/fhirtypes/ElementDefinition.ts

```diff
@@ -524,6 +525,34 @@ export class ElementDefinition {
     }
   }
 
+  /**
+   * Sets flags on this element as specified in a profile or extension.
+   * @todo Add more complete enforcement of rules regarding when these flags can change.
+   * @see {@link http://hl7.org/fhir/R4/profiling.html#mustsupport}
+   * @see {@link http://hl7.org/fhir/R4/elementdefinition-definitions.html#ElementDefinition.mustSupport}
+   * @see {@link http://hl7.org/fhir/R4/elementdefinition-definitions.html#ElementDefinition.isSummary}
+   * @see {@link http://hl7.org/fhir/R4/elementdefinition-definitions.html#ElementDefinition.isModifier}
+   * @param mustSupport - whether to make this element a Must Support element
+   * @param summary - whether to include this element when querying for a summary
+   * @param modifier - whether this element acts as a modifier on the resource
+   * @throws {DisableFlagError} when attempting to disable a flag that cannot be disabled
+   */
+  applyFlags(mustSupport: boolean, summary: boolean, modifier: boolean): void {
+    const disabledFlags = [];
+    if (this.mustSupport && !mustSupport) {
```

Comment:
If `mustSupport` is `null`, I would think that _should_ mean to leave it as-is; but I think this logic would treat it as if it's trying to change the value of `mustSupport` and issue an error.  I think we should check explicitly for `false` here (e.g., `mustSupport === false`).